docs/src/main/sphinx/admin/properties-optimizer.rst

```diff
@@ -116,3 +116,14 @@ the maximum number of joins that can be reordered at once.
 
 Reduces number of rows produced by joins when optimizer detects that duplicated
 join output rows can be skipped.
+
+``optimizer.plan-with-table-node-partitioning``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``true``
+
+Use connector provided table node partitioning when reading tables.
```

Comment:
What is "connector provided table node partitioning" ?
Maybe a concrete example of a connector would help